packages/aws-cdk-lib/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts

```diff
@@ -152,4 +177,14 @@ export class ApplicationLoadBalancedFargateService extends ApplicationLoadBalanc
       throw new Error(`${name}: Must be a non-negative integer; received ${value}`);
     }
   }
+
+  /**
+   * Throws an error if the specified value is greater than the limit.
+   */
+  private validateNotExceeding(name: string, limit: number, value?: number) {
+    if (value === undefined || Token.isUnresolved(value)) { return; }
+    if (value > limit) {
+      throw new Error(`${name}: Must be less than or equal to ${limit}; received ${value}`);
+    }
+  }
```

Comment:
Since this is abstract (I don't think it is good to use it for properties other than cpu and memory), it would be better to make the methods specific to cpu and memory, as described above.

Also, if specifying containerCpu without cpu, what will happen in Cfn deploy?

Reply:
When containerCpu is specified to the same value as cpu, does CFn not cause an error?

Reply:
Yes, I didn't get an error in this case.
(The same was true for memory)